var/spack/repos/builtin/packages/openfdtd/package.py

```diff
@@ -0,0 +1,56 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Openfdtd(MakefilePackage):
+    """OpenFDTD is general purpose FDTD simulator applicable to a wide range
+       of applications. The FDTD method (Finite Difference Time Domain method)
+       is a method for numerically calculating the Maxwell equation, which is
+       the basic equation of the electromagnetic field,
+       by the difference method."""
+
+    homepage = "http://www.e-em.co.jp/OpenFDTD/"
+    url      = "http://www.e-em.co.jp/OpenFDTD/OpenFDTD.zip"
+
+    version('2.3.0', sha256='10ac70f2ed7160da87dd9222a5a17ca7b72365ee886235359afc48c4fb7b4be4')
+
+    variant('mpi', default=False, description='Build with MPI Support')
+
+    depends_on('mpi', when='+mpi')
+
+    def edit(self, spec, prefix):
+        if '%gcc' or '%fj' in self.spec:
```

Comment:
@Hironori-Yamaji does this package build with compilers other than Fujitsu or GCC?

If so, I think

```
filter_file('gcc', spack_cc, './src/Makefile_gcc')
```

should always be run.

Reply:
@scheibelp 
Thank you for your reply!

I agree with you.
There are cases in which this package build with compilers other than Fujitsu or GCC.
Therefore, I changed 
```
filter_file('gcc', spack_cc, './src/Makefile_gcc')
``` 
to always be run.